cmake/Modules/DeployQt5.cmake

```diff
@@ -1,355 +0,0 @@
-# Borrowed from Avogadro project (https://github.com/OpenChemistry/avogadroapp)
```

Comment:
Can't removing of this file and related changes be trivially extracted into separate PR?

Reply:
@glassez 

IMO, if we keep it in this PR, we signal the intent that it is used/will be used. Since it is no longer used, from here on out, it is also removed at this point. It should definitely be its own separate commit though.

Reply:
>if we keep it in this PR, we signal the intent that it is used/will be used

When I suggest separating something from this PR into a separate one, I assume that it should be reviewed and merged before this one.

Reply:
@glassez 

This change does not need to be further reviewed separately, _because it has already been reviewed_. Everyone agreed with this change in the previous version of this PR. The new way of creating the macOS bundle doesn't need this file at all. It has even been previously tested by Kolcha before and verified to work (you can also verify it yourself by inspecting the result of this build: https://github.com/FranciscoPombal/qBittorrent/actions/runs/223928661).

This file was only needed because of the `install_qt5_executable` function, which was previously only called once by `qBittorrent/cmake/Modules/bundle.cmake`, which in turn was a snippet of code called only once from here:

https://github.com/qbittorrent/qBittorrent/blob/defdd5121c940dfc3b807401338e15f66adfd671/src/app/CMakeLists.txt#L161